mdata/notifierKafka/notifierKafka.go

```diff
@@ -128,6 +128,10 @@ func (c *NotifierKafka) start() {
 			}
 		}
 
+		if startOffset < 0 {
+			log.Infof("kafka-cluster: empty partition, nothing to consume")
+			continue
```

Comment:
if we do that, we're not going to consume it if data gets pushed into this partition in the future

Reply:
Please ignore my complete dumb-ness. I'll just fix the `lastReadOffset` in `consumePartition` like I originally said.